test/integration/scenarios/faultTests/solution/update/verify/manifest_test.go

```diff
@@ -0,0 +1,353 @@
+				continue
+			}
+
+			for i := 0; i < 10; i++ {
+				err = shellcmd.Command(fmt.Sprintf("kubectl apply -f %s -n %s", fullPath, namespace)).Run()
+				if err == nil {
+					break
+				}
+				time.Sleep(5 * time.Second)
+			}
+			require.NoError(t, err)
+		}
+		// Sleep for 10 second to hit the failure point
+		time.Sleep(10 * time.Second)
+
+		verifyTargetStatus(t, test, namespace)
+		verifyInstanceStatus(t, test, namespace)
+		verifyPodsExist(t, test, test.PodsToVerify)
+		verifyPodsDeleted(t, test, test.DeletedPodsToVerify)
+	}
+}
+
+// Verify target has correct status
+func verifyTargetStatus(t *testing.T, test TestCase, namespace string) {
+	// Verify targets
+	crd := &unstructured.Unstructured{}
+	crd.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "fabric.symphony",
+		Version: "v1",
+		Kind:    "Target",
+	})
+
+	cfg, err := testhelpers.RestConfig()
+	require.NoError(t, err)
+
+	dyn, err := dynamic.NewForConfig(cfg)
+	require.NoError(t, err)
+
+	for {
+		resources, err := dyn.Resource(schema.GroupVersionResource{
+			Group:    "fabric.symphony",
+			Version:  "v1",
+			Resource: "targets",
+		}).Namespace(namespace).List(context.Background(), metav1.ListOptions{})
+		require.NoError(t, err)
+
+		require.Len(t, resources.Items, 1, "there should be only one target")
+
+		status := getStatus(resources.Items[0])
+		fmt.Printf("Current target status: %s\n", status)
+		require.NotEqual(t, "Failed", status, fmt.Sprintf("%s: Target should not be in failed state", test.Name))
+		if status == "Succeeded" {
+			break
+		}
+
+		sleepDuration, _ := time.ParseDuration("30s")
+		time.Sleep(sleepDuration)
+	}
+}
+
+// Verify instance has correct status
+func verifyInstanceStatus(t *testing.T, test TestCase, namespace string) {
+	// Verify instances
+	cfg, err := testhelpers.RestConfig()
+	require.NoError(t, err)
+
+	dyn, err := dynamic.NewForConfig(cfg)
+	require.NoError(t, err)
+
+	for {
+		resources, err := dyn.Resource(schema.GroupVersionResource{
+			Group:    "solution.symphony",
+			Version:  "v1",
+			Resource: "instances",
+		}).Namespace(namespace).List(context.Background(), metav1.ListOptions{})
+		require.NoError(t, err)
+
+		require.Len(t, resources.Items, 1, "there should be only one instance")
+
+		status := getStatus(resources.Items[0])
+		fmt.Printf("Current instance status: %s\n", status)
+		require.NotEqual(t, "Failed", status, fmt.Sprintf("%s: Instance should not be in failed state", test.Name))
+		if status == "Succeeded" {
+			break
+		}
+
+		sleepDuration, _ := time.ParseDuration("30s")
+		time.Sleep(sleepDuration)
+	}
+}
+
+// Verify that the pods we expect are running in the namespace
+// Lists pods from the cluster and then verifies that the
+// expected strings are found in the list.
+func verifyPodsExist(t *testing.T, test TestCase, toFind []string) {
+	// Get kube client
+	kubeClient, err := testhelpers.KubeClient()
+	require.NoError(t, err)
+
+	i := 0
+	for {
+		i++
```

Comment:
`i` can be changed to a local variable with a scope of for-loop
```go
for i := 0; i++; {}
```

Reply:
I don't think it is allowed without a condition. Could you please try?